tests/unit/lms/resources/_js_config_test.py

```diff
@@ -91,17 +103,14 @@ def test_it_includes_grant_token(self, config):
         assert before <= claims["nbf"] <= after
         assert claims["exp"] > before
 
-    def test_it_includes_the_group(self, pyramid_request, config):
+    def test_it_includes_the_group(self, config):
         groups = config["services"][0]["groups"]
 
         assert groups == ["example_groupid"]
 
-    def test_it_is_empty_if_provisioning_feature_is_disabled(
-        self, context, pyramid_request
-    ):
-        context.provisioning_enabled = False
-
-        assert JSConfig(context, pyramid_request).config["hypothesisClient"] == {}
+    @pytest.mark.usefixtures("provisioning_disabled")
+    def test_it_is_empty_if_provisioning_feature_is_disabled(self, config):
+        assert config == {}
```

Comment:
The `provisioning_disabled` fixture is only used once (so far) but it's still worth moving it out into a fixture for how much it simplifies this test and makes this test more consistent with the other tests, by letting it use the `config` fixture instead of initializing its own `JSConfig`

Reply:
I like this, it's nice and declarative. 

You also know that the fixture is supposed to _do_ something or setup some condition, rather than _be_ something. This generally makes it easier to understand, because you can trust it does what it says on the tin and carry on. With a fixture, you usually have to go read it to work out what it is, and if it changes anything.